tests/test_browser.py

```diff
@@ -3046,7 +3075,12 @@ def test_emterpreter_async(self):
       self.btest('emterpreter_async.cpp', '1', args=['-s', 'EMTERPRETIFY=1', '-s', 'EMTERPRETIFY_ASYNC=1', '-O' + str(opts), '-g2'])
 
   def test_emterpreter_async_2(self):
-    self.btest('emterpreter_async_2.cpp', '40', args=['-s', 'EMTERPRETIFY=1', '-s', 'EMTERPRETIFY_ASYNC=1', '-O3'])
+    # Error.stackTraceLimit default to 10 in chrome by this test relies on more
```

Comment:
"default to 10 in chrome by this test" is confusing to me.

Oh, typo: s/by/but/